CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [7.0.3] - 2020-07-23
+
 ### Changed
 
 - Change etcd data dir permission to `0700` to comply with etcd 3.10.4 requirements.
```

Comment:
```suggestion
### Fixed

- Set etcd data dir permission to `0700` to comply with etcd 3.10.4 requirements.
```

Reply:
somehow I cannot use the suggested change on github :(